site/_resources/envoy.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Envoy compatibility matrix
+layout: page
+---
+
+Due to the agressive deprecation cycle of Envoy's xDS API, not all versions of Contour will work with all versions of Envoy, and vice versa.
+
+This page describes the compatibility matrix of Contour and Envoy versions.
+
+## Supported Envoy versions
+
+|              | Contour 1.0.0 | 
+| ------------ | :-----------:|
+| Envoy 1.11.0 | Not supported<sup>1</sup> |
+| Envoy 1.11.1 | Not supported<sup>2</sup> |
+| Envoy 1.11.2 |  Supported | 
+
+#### Notes
+
+1. [CVE-2019-9512, CVE-2019-9513, CVE-2019-9514, CVE-2019-9515, CVE-2019-9518][1] 
+2. [CVE-2019-15225, CVE-2019-15226][2]
+
+## Envoy extensions
+
+Contour requires the following extensions.
+If you are using the image recommended in our [example deployment](https://github.com/projectcontour/contour/blog/{{ site.github.latest_release.tag_name }}/examples/contour) no action is required.
+If you are providing your own Envoy it must be compiled with the following extensions:
+
```

Comment:
This ends up with a lot of red text. It's probably impractical to link to Envoy docs for each plugin, so maybe format this as a table?

| Access Loggers |  envoy.file_access_log , envoy.http_grpc_access_log, envoy.tcp_grpc_access_log |

Reply:
I thought about that, but putting it in a table was just too close to being one of those people who keeps lists of things in excel. I felt it was easier to copy and paste using a <li>